cli/src/main/java/org/eclipse/hono/cli/app/Receiver.java

```diff
@@ -99,6 +120,14 @@ private CompositeFuture createConsumer(final HonoConnection connection) {
         return CompositeFuture.all(consumerFutures);
     }
 
+    /**
+     * Handle received messge.
+     * 
+     * Write log messages to stdout.
+     * 
+     * @param endpoint receiving endpont, "telemetry" or "event".
+     * @param msg recevied message
+     */
```

Comment:
Typos (messge, endpont, recevied). Or javadoc can be removed again (method is again private now).